scripts/backup_script.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Stop services that should not run during the backup
+docker compose stop nginx flower worker beat web mq solr
```

Comment:
do we need to stop all the services?

Reply:
to prevent any changes to the data while the backup is being created.

Reply:
The backup was running fine so far with the service running. Not that a short downtime is an issue.
We can discuss this with @neoflex  :)